src/uitpas/search/detail/UiTPasAdvantageDescription.js

```diff
@@ -170,12 +170,7 @@ export default class UiTPasAdvantageDescription extends React.Component {
       this.isOnlineExchangeable()
     ) {
       return (
-        <a
-          href={this.onlineExchangeUrl()}
-          target="_blank"
-          rel="noopener"
-          className="sk-button"
-        >
+        <a href={this.onlineExchangeUrl()} rel="noopener" className="sk-button">
```

Comment:
I think the `rel="noopener"` can be removed now as well, it's only relevant for links that open in a new tab if I'm not mistaken.

Reply:
correct